src/main/java/io/openapitools/swagger/OutputFormat.java

```diff
@@ -39,9 +40,11 @@ public void write(OpenAPI swagger, File file) throws IOException {
     static class JSONWriter implements SwaggerWriter {
 
         @Override
-        public void write(OpenAPI swagger, File file) throws IOException {
-            ObjectMapper mapper = new ObjectMapper();
-            mapper.setSerializationInclusion(JsonInclude.Include.NON_EMPTY);
+        public void write(OpenAPI swagger, File file, boolean prettyPrint) throws IOException {
+            ObjectMapper mapper = Json.mapper();
```

Comment:
Good point, this would ensure we serialize the same way as Swagger core library.